examples/java/veeam-vm-backup/README.MD

```diff
@@ -0,0 +1,144 @@
+git checkout master
+```
+
+## Customize the function
+
+For security reasons to not expose sensitive data we will create a Kubernetes secret (https://kubernetes.io/docs/concepts/configuration/secret/) which will hold Veeam credentials and information, as well as the vCenter ip . This secret will be mounted into the function during runtime. This is all taken care of for your by the appliance. We only have to create the secret with a simple command through `faas-cli`.
+
+First, change the configuration file `veeamconfig.toml` holding your secret vCenter information located in the veeam-vm-backup example folder:
+
+```toml
+# veeamconfig.toml contents
+# replace with your own values
+[enterprise_manager]
+ip = "VEEAM_ENTERPRISE_MANAGER_IP"
+port = "9398"	# This is the default port for Veeam's Restful API. Change it, if you changed the port during Veeam configuration. 
+user = "USERNAME"
+password = "PASSWORD"
+
+[backup_server]
+id = "BACKUP_SERVER_ID"  # Can be found through the request to your Veeam server described here: https://helpcenter.veeam.com/docs/backup/rest/get_backupservers.html?ver=100
+
+[vcenter]
+ip = "VC_IP"
+```
+
+Now go ahead and store this configuration file as secret in the appliance.
+
+```bash
+# set up faas-cli for first use
+export OPENFAAS_URL=https://VEBA_FQDN_OR_IP
+faas-cli login -p VEBA_OPENFAAS_PASSWORD --tls-no-verify # vCenter Event Broker Appliance is configured with authentication, pass in the password used during the vCenter Event Broker Appliance deployment process
+
+# now create the secret
+faas-cli secret create veeamconfig --from-file=veeamconfig.toml --tls-no-verify
+```
+
+**Note:** Delete the local `veeamconfig.toml` after you're done with this exercise to not expose this sensitive information.
+
+Lastly, define the function-specific settings. They are performed in the `veeam-vm-backup.yml` file. Open and edit the `veeam-vm-backup.yml` provided with in the Java veeam-vm-backup example code. Change `gateway` and (optionally) `topic` as per your environment/needs.
+
+```yaml
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP # replace with your vCenter Event Broker Appliance environment
+functions:
+  veeam-vm-backup:
+    lang: java8
+    handler: ./vm-backup
+    image: vmware/veeam-vm-backup
+    secrets:
+      - veeamconfig # leave as is unless you changed the name during the creation of the secrets above
+    annotations:
+      topic: VmPoweredOffEvent  # the event can be changed, in case the aim is to back up a VM when another type of event occurs
+```
+
+
+## Deploy the function
+
+After you've performed the steps and modifications above, you can go ahead and deploy the function:
+
+```bash
+faas-cli template pull # only required during the first deployment
+# Get the template directories that veeam-vm-backup needs. Again, only required during the first deployment
+cp -r  ./template/java8/gradle ./vm-backup
```

Comment:
Is this a special case for the Java template where you have to copy around content from `template`?

Reply:
This part is removed from the edited README.md file, as these steps were not needed for the successful deployment of the function.